server/src/ClientToDBAPI/ClinicalDetailsRoute/mainRoute.ts

```diff
@@ -1,9 +1,58 @@
 import { Router, Request, Response } from 'express';
 
+import { graphqlRequest } from '../../GraphqlHTTPRequest';
+import { CREATE_ADDRESS, CREATE_INVESTIGATION, ADD_BACKGROUND_DESEASES, ADD_SYMPTOMS } from '../../DBService/ClinicalDetails/Mutation';
+
 const clinicalDetailsRoute = Router();
 
 clinicalDetailsRoute.get('/', (request: Request, response: Response) => {
     response.send('Hello from Clinical Details route');
-})
+});
+
+clinicalDetailsRoute.post('/saveClinicalDetails', (request: Request, response: Response) => {
+    
+    const requestAddress = request.body.input;
+
+    const requestInvestigation = request.body.investigation;
+    
+    graphqlRequest(CREATE_ADDRESS, { input: {
+        ...requestAddress
+    }}).then((result: any) => {
```

Comment:
add type

Reply:
it has no type. each time its diffrent

Reply:
how is this possible, you are always returning the same structure

Reply:
you are right, but i dont think we need to create a type for everything. Just like we dont usually create an interface for every data we get from useQuery (we only create interfaces for the actual data). Ata mevin ma ani omer? 
I mean i can create an interface but is it necessary?

Reply:
i think you can do it (I did it in the axios request)

Reply:
i will check